pytensor/tensor/basic.py

```diff
@@ -3084,87 +3084,72 @@ def flatten(x, ndim=1):
     return x_reshaped
 
 
-def tile(x, reps, ndim=None):
+def tile(
+    A: "TensorLike", reps: Union[Sequence[int, "TensorLike"], "TensorLike"]
+) -> TensorVariable:
     """
-    Tile input array `x` according to `reps`.
+    Tile input array `A` according to `reps`.
 
     See the docstring of `numpy.tile` for details.
 
-    'reps' can be constant integer (e.g. 3), constant vector(e.g. [2 3]),
-    symbolic scalar (e.g. tensor.iscalar()), symbolic vector (e.g. tensor.ivector())
-    or a list of symbolic scalar (e.g. [tensor.iscalar(), tensor.iscalar()]).
-
-    ndim is the number of the dimensions of the output, if it is provided, ndim
-    should be equal or larger than x.ndim and len(reps), otherwise, we will use
-    max(x.ndim, len(reps)) as ndim. If reps is symbolic vector, the ndim has to
-    be provided.
-
+    If `reps` is a PyTensor vector, it's length must be statically known.
+    You can use `specify_shape` to set the length.
```

Comment:
Would it be possible to give an example? I think it's difficult to understand how to do that if you don't know `specify_shape`

Reply:
Srue